### PR TITLE
Updated Readme to make it more clear.

### PR DESCRIPTION
**Pull request by arunpatyal**
1. Changes as per review.
2. Also renamed a few variables to make the code consistent to use Jira Server everywhere instead of Jira Onprem.

### CODE REVIEW

File: atlassian-terraform/atlassian.auto.tfvars [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Sumo Logic - Atlassian Terraform
+# Configure Jira Cloud, Jira Server, OpsGenie, BitBucket credentials and parameters.
+
+
+# Jira Cloud
+jira_cloud_url      = "https://arunpatyal.atlassian.net"
+jira_cloud_user     = "arunpatyal0809@gmail.com"
+jira_cloud_password = "AVQYnQwD1IdI2GylyUQx99B6" # Password or API Key, To generate API Key: https://confluence.atlassian.com/cloud/api-tokens-938839638.html
```
Comment (duchatran) [inline atlassian-terraform/atlassian.auto.tfvars:8]: Wow, need to change this immediately

File: atlassian-terraform/atlassian.auto.tfvars [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+"jira_expression_evaluation_failed"]
+
+
+# Jira Server
+# This script configures Jira Server WebHooks. Jira Server Logs collection needs to be configured as explained in Step 1 [here](https://help.sumologic.com/07Sumo-Logic-Apps/08App_Development/Jira/Collect_Logs_for_Jira#step-1-set-up-local-file-sources-on-an-installed-collector).
+# Configure the log collection and update the variable jira_server_access_log_category with the selected source category.
+jira_server_access_logs_sourcecategory = "Atlassian/Jira/Server*"
+jira_server_url                        = "http://localhost:8080"
+jira_server_user                       = "arunpatyal0809"
+jira_server_password                   = "Qazwsx@2!" # Needs to be the password. API Key is not supported on Jira Server yet: https://jira.atlassian.com/browse/JRASERVER-67869?_ga=2.198461357.302520551.1583314185-1454539139.1580206139&error=login_required&error_description=Login+required&state=d3142ec3-6eb1-4207-bdd7-ce6b93900aa1
```
Comment (duchatran) [inline atlassian-terraform/atlassian.auto.tfvars:30]: ANother password leak, but not as important unless you reuse.

File: atlassian-terraform/atlassian.auto.tfvars [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  "user_created", "user_updated", "user_deleted",
+  "option_voting_changed", "option_watching_changed", "option_unassigned_issues_changed", "option_subtasks_changed", "option_attachments_changed", "option_issuelinks_changed", "option_timetracking_changed",
+  "sprint_created", "sprint_deleted", "sprint_updated", "sprint_started", "sprint_closed",
+"board_created", "board_updated", "board_deleted", "board_configuration_changed"]
+
+
+# Bitbucket Cloud
+# Parameter Description: https://www.terraform.io/docs/providers/bitbucket/r/hook.html
+bitbucket_cloud_user     = "arunpatyalsumo"
+bitbucket_cloud_password = "PeJ6NrdjQfRdR8u7zvYD"      # App Passwords also work. To generate App Password see: https://confluence.atlassian.com/bitbucket/app-passwords-828781300.html
```
Comment (duchatran) [inline atlassian-terraform/atlassian.auto.tfvars:46]: another password

File: atlassian-terraform/sumologic.auto.tfvars [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Sumo Logic - Atlassian Terraform
+# Configure Sumo Logic credentials and App installation options.
+
+# Sumo Logic
+sumo_access_id          = "suZGHyWOCpae0J" # https://help.sumologic.com/Manage/Security/Access-Keys
+sumo_access_key         = "Cp4PFmcX95yMFQOQctToPjGwoYpQcy8YGUN4ZsbLUqFsm6ZiSW8zVrznOhZX8MGh"
```
Comment (duchatran) [inline atlassian-terraform/sumologic.auto.tfvars:6]: Another password - need to retire this keypair

File: atlassian-terraform/sumologic.auto.tfvars [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+deployment              = "us1"                                    # https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
+sumo_api_endpoint       = "https://nite-api.sumologic.net/api/v1/" # https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
```
Comment (duchatran) [inline atlassian-terraform/sumologic.auto.tfvars:8]: Don't expose internal endpoints.

File: atlassian-terraform/webhooks.auto.tfvars [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+# Sumo Logic to Opsgenie Webhook
+opsgenie_priority = "P3"
+
+
+# Sumo Logic to Jira Cloud Webhook - This feature is in Beta. To participate contact your Sumo account executive.
+# https://developer.atlassian.com/cloud/jira/platform/rest/v2/#api-rest-api-2-issue-post
+jira_cloud_issuetype  = "Bug"
+jira_cloud_priority   = "3"
+jira_cloud_projectkey = "WOZ"
+jira_cloud_auth       = "Basic YXJ1bnBhdHlhbDA4MDlAZ21haWwuY29tOkFWUVluUXdEMUlkSTJHeWx5VVF4OTlCNg==" # Basic Authorization Header. See: https://help.sumologic.com/Beta/Webhook_Connection_for_Jira_Cloud#prerequisite
```
Comment (duchatran) [inline atlassian-terraform/webhooks.auto.tfvars:14]: Another sensitive cred leak?

File: atlassian-terraform/webhooks.auto.tfvars [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+
+# Sumo Logic to Jira Server Webhook - This feature is in Beta. To participate contact your Sumo account executive.
+# https://docs.atlassian.com/software/jira/docs/api/REST/7.6.1/#api/2/issue-createIssue
+jira_server_issuetype  = "Bug"
+jira_server_priority   = "3"
+jira_server_projectkey = "WOZ"
+jira_server_auth       = "Basic YXJ1bnBhdHlhbDA4MDk6UWF6d3N4QDIh" #Basic Authorization Header. See: https://help.sumologic.com/Beta/Webhook_Connection_for_Jira_Server#prerequisite
```
Comment (duchatran) [inline atlassian-terraform/webhooks.auto.tfvars:22]: Another sensitive cred?

File: atlassian-terraform/atlassian.auto.tfvars (hunk 1 above)
Comment (duchatran) [reply atlassian-terraform/atlassian.auto.tfvars:8]: replace with things like "me@example.com", "mycloudpassword"

File: atlassian-terraform/atlassian.auto.tfvars [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+bitbucket_cloud_owner    = "arunpatyalsumo"            # The owner of this repository. Can be you or any team you have write access to.
```
Comment (ankitgoelcmu) [inline atlassian-terraform/atlassian.auto.tfvars:47]: another one

File: atlassian-terraform/atlassian.auto.tfvars (hunk 1 above)
Comment (arunpatyal) [reply atlassian-terraform/atlassian.auto.tfvars]: Wrong file was uploaded. Correct file with placeholders has been uploaded and all the credentials are revoked.

File: atlassian-terraform/atlassian.auto.tfvars (hunk 2 above)
Comment (arunpatyal) [reply atlassian-terraform/atlassian.auto.tfvars]: Wrong file was uploaded. Correct file with placeholders has been uploaded and all the credentials are revoked.

File: atlassian-terraform/atlassian.auto.tfvars (hunk 3 above)
Comment (arunpatyal) [reply atlassian-terraform/atlassian.auto.tfvars]: Wrong file was uploaded. Correct file with placeholders has been uploaded and all the credentials are revoked.

File: atlassian-terraform/sumologic.auto.tfvars (hunk 5 above)
Comment (arunpatyal) [reply atlassian-terraform/sumologic.auto.tfvars]: Wrong file was uploaded. Correct file with placeholders has been uploaded and all the credentials are revoked.

File: atlassian-terraform/webhooks.auto.tfvars (hunk 6 above)
Comment (arunpatyal) [reply atlassian-terraform/webhooks.auto.tfvars]: Wrong file was uploaded. Correct file with placeholders has been uploaded and all the credentials are revoked.

File: atlassian-terraform/atlassian.auto.tfvars (hunk 8 above)
Comment (arunpatyal) [reply atlassian-terraform/atlassian.auto.tfvars]: Wrong file was uploaded. Correct file with placeholders has been uploaded and all the credentials are revoked.

File: atlassian-terraform/webhooks.auto.tfvars (hunk 7 above)
Comment (arunpatyal) [reply atlassian-terraform/webhooks.auto.tfvars]: Fixed

**Pull request merged**